information.txt

```diff
@@ -48,7 +52,7 @@ Overall evaluation: 2 (accept)
 ----------- Overall evaluation -----------
 The paper is about the experience of the authors in organizing a workshop focused on the analysis and interpretation of neu- roimaging data, ranging from whole-brain functional magnetic res- onance imaging (fMRI) to single-cell microscopy.
 
-They used a cloud computing services and advanced cyberinfrastructure support for improving learning ex-periences in bootcamp-style events. I found the paper interesting, although it should be more general (some of the proposed solutions are specific for their campus).
+They used a cloud computing services and advanced cyberinfrastructure support for improving learning ex-periences in bootcamp-style events. I found the paper interesting, although it should be more general (some of the proposed solutions are specific for their campus).  <--- Hopefully addressed by the paragraph on the XSEDE cluster being widely available etc.
```

Comment:
I am curious how their reading of the paper came to the conclusion of being very campus-specific, since docker+XSEDE is about as generalized as you can get within the research computing space since the XSEDE resources are available to all US-based researchers.